src/DB_Command.php

```diff
@@ -258,6 +261,8 @@ public function check( $_, $assoc_args ) {
 
 		WP_CLI::debug( 'Associative arguments: ' . json_encode( $assoc_args ), 'db' );
 		WP_CLI::success( 'Database checked.' );
+		WP_CLI::log( 'Check whether the WordPress DB tables are installed: wp core is-installed', 'db' );
+		WP_CLI::log( 'Check whether the WordPress DB tables need to be updated: wp core update-db --dry-run', 'db' );
```

Comment:
Do we want to wrap these in a conditional that checks for success from the `db check` command?

Reply:
Updated at https://github.com/wp-cli/db-command/pull/244/commits/7923571db2fa2355174cf23358eed8859bd1cf07